mule-extensions-api-persistence/src/test/java/org/mule/runtime/extension/api/persistence/ModelPropertyPersistenceTestCase.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.persistence;
+
+import static java.util.Collections.singletonMap;
+
+import org.mule.runtime.extension.api.introspection.property.ImportedTypesModelProperty;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.TypeAdapter;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+
+import org.junit.Test;
+
+public class ModelPropertyPersistenceTestCase extends BasePersistenceTestCase
```

Comment:
this is a class to test all the custom serialisers for model properties, now this is the unique one